pkgs/by-name/xr/xrock/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  fetchFromGitHub,
+  lib,
+  libusb1,
+  pkg-config,
+  stdenv,
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Let's use the `finalAttrs` pattern here, to have better overrid-ability, see:
- #315337